.gitignore

```diff
@@ -15,6 +15,7 @@ user_playbooks
 playbooks/galaxy_roles
 requirements.yml
 inventories/local_*
+user_devel_env_files/*
```

Comment:
I saw this is included but with a .gitkeep... I think I am less inclined to cloud the top level _committed_ directories to this repository and don't consider it too much to ask a developer to simply create this folder. If we do keep it, maybe shorten it down to `user_env_files` given this is applicable for use on any machine and any setup.

Reply:
The example would be, I sometimes want to setup `root` home for some activities with some of my `vim` configuration.

Reply:
sure, we can keep it uncommitted and ask the user to create it